src/job.rs

```diff
@@ -123,6 +123,12 @@ pub struct Job {
     pub pending_error: Option<String>,
 }
 
+impl fmt::Display for Job {
```

Comment:
isn't Debug more appropriate for this?

Reply:
Debug for Job structure is already derived and it shows all the fields. It is probably useful to keep it this way and use it for quick printf-debugging.